gptel.el

```diff
@@ -372,6 +372,9 @@ This is an alist mapping major modes to the reply prefix strings.  This
 is only inserted in dedicated gptel buffers before the AI's response."
   :type '(alist :key-type symbol :value-type string))
 
+(defcustom gptel-response-separator "\n\n"
+  "String inserted before responseses.")
```

Comment:
`responseses`: intentional?

Reply:
I was kidnapped by people who live in row-houses in Brooklyn.